### PR TITLE
create Topics table and insert topics to it

### LINKED ISSUE #23: DB: Create a migration and initial seed for the Topics table
Create a Knex migration for the Topics table. Also, consider adding a seed file for development to add a couple of initial topics.

image

### PR DESCRIPTION
**Pull request by nuha90**
# Description

 Knex migration for the Topics table was created. Also,couple of initial topics. was added to the seed file

Fixes #23 
Create a migration and initial seed for the Topics table 

# Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -24,4 +24,4 @@ OPEN_BROWSER="true"
 # DEBUG=knex:query
 
 # Node environment set to either "development" or "production"
-NODE_ENV=development
+NODE_ENV=development
```
Comment (tpetrina) [inline .env.example:27]: Hit enter after this line and commit the file again.

File: src/server/migrations/20200510015322_create_modules_schema.js [hunk 2]
```diff
@@ -1,24 +1,12 @@
 // TODO: This is a sample migration for demonstration. Remove before adding real ones.
 
 exports.up = function(knex) {
-  return knex.schema.createTable('modules', (table) => {
+  return knex.schema.createTable('Topics', (table) => {
```
Comment (tpetrina) [inline src/server/migrations/20200510015322_create_modules_schema.js:4]: Please use lowercase name `topics` for consistency.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Good work, small fixes needed.

File: src/server/migrations/20200510015322_create_modules_schema.js (hunk 2 above)
Comment (nuha90) [reply src/server/migrations/20200510015322_create_modules_schema.js]: updated toni

**Review by tpetrina — DISMISSED** [review]
Good job.

**Review by claudion3 — APPROVED** [review]
well done

File: src/server/seeds/development/topics.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+// TODO: This is a sample seed file for demonstration. Remove before adding real ones.
```
Comment (tpetrina) [inline src/server/seeds/development/topics.js:1]: Remove this comment.

File: src/server/migrations/20200510015322_create_topics_schema.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// TODO: This is a sample migration for demonstration. Remove before adding real ones.
```
Comment (tpetrina) [inline src/server/migrations/20200510015322_create_topics_schema.js:1]: Remove this comment.

**Review by MaherHussain — COMMENTED** [review]
great

**Comment by dpfernandes** [comment]
Good job but we are going to handle the Topics table on this PR https://github.com/HackYourFuture-CPH/web-dev-rooms/pull/69/

**Pull request closed without merging**